scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/hvssop.yaml

```diff
@@ -0,0 +1,55 @@
+FileHeader:
+  library_Suffix: 'SO'
+  device_type: 'HVSSOP'
+
+HVSSOP-10-1EP_3x3mm_P0.5mm_EP1.57x1.88mm:
+  size_source: 'https://www.ti.com/lit/ds/symlink/bq24090.pdf'
+
+  body_size_x:
+    nominal: 3
+    tolerance: 0.100
+  body_size_y:
+    nominal: 3
+    tolerance: 0.100
+  overall_height:
+    maximum: 1.1
+
+  overall_size_x:
+    nominal: 4.9
+    tolerance: 0.152
```

Comment:
I see 5.05 .. 4.75 No clue where the 0.152 value comes from.
You can just enter them as
```yaml
minimum 4.75
maximum 5.05
```

Reply:
Done